reinforcement_learning/rl_cartpole_coach/rl_cartpole_coach_gymEnv.ipynb

```diff
@@ -4,12 +4,15 @@
    "cell_type": "markdown",
```

Comment:
<p>Cart-pole Balancing Model with Amazon SageMaker and Coach<strong> Library</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added.